web/src/tensil/tools/web/Main.scala

```diff
@@ -32,11 +32,11 @@ object CompilerTask {
   implicit val rw: ReadWriter[CompilerTask] = macroRW
 
   // TODO: these queue, table and bucket names to be configured from container environment
-  val queue          = sqs.queue("tensil-web-compiler-7af9ea2").get
-  val jobTable       = db.table("tensil-web-job-c9cb5e4").get
-  val taskTable      = db.table("tensil-web-task-731b0cb").get
-  val compilerBucket = s3.bucket("tensil-web-compiler-8681a04").get
-  val uploadsBucket  = s3.bucket("tensil-web-upload-7a8bd18").get
+  val queue          = sqs.queue("tf2rtl-web-compiler-7af9ea2").get
```

Comment:
Can we drop a TODO to change these services names back to tensil?

Reply:
It can be somewhat tedious as we'll need to backup data from bunch of Dynamo tables and S3 buckets and then restore. I think we'd just move this from code to configuration and treat these object names as opaque IDs.